### ISSUE TITLE
Shutting down Windows while Chrome is open causes open tabs to show as "recovered"

### ISSUE THREAD

**Issue by cxw42** [cxw42/TabFern#55]
Apparently, the normal close processing doesn't happen when the browser is shutting down.  See [pertinent bug report](https://bugs.chromium.org/p/chromium/issues/detail?id=30885).  Observed in v0.1.7.

**Comment by bluikko** [comment]
Great results! Thank you!

On 0.1.15, after clicking "Restart" for Windows Updates, all windows/tabs were up-to-date and kept their names on Chrome start after reboot.

However:
After opening a window called "MAINLINE", now I have both window called "MAINLINE" (in closed state in TabFern) and "Unsaved" with the same tabs.
I rarely close Chrome so it might be a few weeks until I can test again. I will open a new issue if I can confirm what I did exactly. I was not thinking much, this time, when clicking to re-open the windows.

**Comment by cxw42** [comment]
@bluikko yes, please open a new issue if you experience the problem again.  This issue is for shutting down Windows without closing all your open tabs first.

For what it's worth, the automatic merge currently only works for actions taken by the system.  If you interact with the TabFern window manually, it assumes you are doing what you want :) .  I was concerned that, for example, if you happened to manually open a tab set that matched a closed window way down in the tree, a sudden merge might be unexpected.  As always, I am happy to take feature requests if there is specific behaviour you would like.

**Comment by thor79** [comment]
When I close chrome using the exit command in the chrome menu, upon reopening it, all my pinned tabs appear in a single window (from multiple windows). All non-pinned tabs are stored under "recovered tabs". I frequently have many non-pinned tabs in suspended mode (suspended by The Great Suspender) so I leave a lot of tabs open for sites I may visit somewhat frequently or are associated with things I'm currently interested in.

**Comment by cxw42** [comment]
@thor79 Good to hear from you!  I have never used Great Suspender, although I hear good things about it.    I will open a separate issue since the issue happens on exit and not just on shutdown.

I just checked the upstream bugs noted above, and there hasn't been any reported progress.  Therefore, TabFern's ability to properly handle any Chromium shutdown will continue to be somewhat limited :/ .
